doc/reduce-memory.md

```diff
@@ -24,7 +24,9 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 ## Number of peers
 
-- `-maxconnections=<n>` - the maximum number of connections, this defaults to `125`. Each active connection takes up some memory. Only significant if incoming connections are enabled, otherwise the number of connections will never be more than `10`. Of the 10 outbound peers, there can be 8 full outgoing connections and 2 -blocksonly peers, in which case they are block/addr peers, but not tx peers.
+- `-maxconnections=<n>` - the maximum number of connections, this defaults to `125`. Each active connection takes up
+  some memory. Only significant if incoming connections are enabled, otherwise the number of connections will never be
```

Comment:
suggest clarifying with s/Only significant/This option applies only/ or something similar

(as is, it reads almost like the memory use is only significant if...)